crates/router/src/connector/aci/transformers.rs

```diff
@@ -91,9 +91,37 @@ impl TryFrom<&api_models::payments::WalletData> for PaymentDetails {
                     account_id: None,
                 }))
             }
-            _ => Err(errors::ConnectorError::NotImplemented(
-                "Payment method".to_string(),
-            ))?,
+            api_models::payments::WalletData::MbWayRedirect(_)
```

Comment:
Clippy checks are failing as MbWayRedirect is already handled above in line 82,
So remove this line(94)